script/Arcueid/Day 8/08_00_ARC08_3_1.txt

```diff
@@ -269,7 +269,7 @@ She's as birdbrained as ever.
 [sha:e5bfe93043d3b17b9e72066370b1ac733b6b4aeb]{
 -- Page 178, Offset 20111.
 -- 「……助かった。そんな事されてたら授業どころの話じゃなかった……」
-"...Thank god. If you had done that, then the class would have been the least of my worries..."
+"...Thank goodness. If you had done that, then the class would have been the least of my worries..."
```

Comment:
revert

Reply:
it's a personal preference. everything up to this point has been changed to goodness except when Noel or other church members uses it. is that a bad change?

Reply:
I prefer "thank god" personally and I don't really think drawing a line between church vs. non-church is worth doing.

Reply:
ok sure i'll revert it then

Reply:
capitalize first letter of `god` if so

Reply:
i'm not so sure that's necessary i dont think hes' referring to any god in particular